src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -165,9 +166,18 @@ public function loadClass($class)
 
             if (in_array(strtolower($refl->getShortName()), self::$php7Reserved)) {
                 @trigger_error(sprintf('%s uses a reserved class name (%s) that will break on PHP 7 and higher', $name, $refl->getShortName()), E_USER_DEPRECATED);
-            } elseif (preg_match('#\n \* @deprecated (.*?)\r?\n \*(?: @|/$)#s', $refl->getDocComment(), $notice)) {
+            }
+
+            if (preg_match('#\n \* @final( .*)?\r?\n \*(?: @|/$)#s', $refl->getDocComment())) {
```

Comment:
this does not need to be done for interfaces (marking an interface as final does not make any sense). Add ``$refl->isClass()``